modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -128,7 +128,7 @@ EXTERNAL-IP   OS-IMAGE                                       	KERNEL-VERSION
 CONTAINER-RUNTIME
 rt-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.22.1
 128.66.135.107   <none>    	        Red Hat Enterprise Linux CoreOS 46.82.202008252340-0 (Ootpa)
-4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.1-90.rhaos4.9.git4a0ac05.el8-rc.1
+4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.1-90.rhaos4.10.git4a0ac05.el8-rc.1
```

Comment:
This is not a correct change, you would have to replace the full string here, 4.10 will be based on k8s 1.23 for example.

Reply:
@marsik on a 4.10 nightly I see: 

```
$ oc get node -o wide
NAME                                                     STATUS   ROLES    AGE   VERSION           INTERNAL-IP   EXTERNAL-IP   OS-IMAGE                                                        KERNEL-VERSION                 CONTAINER-RUNTIME
openshift-master-0.openshift.pool2.practice.redhat.com   Ready    master   10d   v1.22.1+6859754   10.1.196.21   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
openshift-master-2.openshift.pool2.practice.redhat.com   Ready    master   10d   v1.22.1+6859754   10.1.196.23   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
openshift-worker-0.openshift.pool2.practice.redhat.com   Ready    worker   10d   v1.22.1+6859754   10.1.196.24   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
openshift-worker-1.openshift.pool2.practice.redhat.com   Ready    master   10d   v1.22.1+6859754   10.1.196.25   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
```
Will `cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8` be accurate for GA?